README.md

```diff
@@ -57,7 +57,9 @@ To run the OAuth example, you will need to have your Kafka cluster configured wi
 ## Configuration
 
 Although this Hello World is simple example it is fully configurable.
-Below are listed and described environmental variables.
+Below are listed and described environmental variables that are NOT Kafka-specific but must be used. 
+These environmental variables should be prefixed with `STRIMZI_` using `_` instead of `.` (eg. STRIMZI_TOPIC). 
```

Comment:
These are just random environment variables. The `_` versus `.` makes no sense here. It makes sense only for the `KAFKA_` env vars which map to actual Kafka configuration properties.

Reply:
Removed and refactored to fit the uniformity of the rest of the README.